calendar-kata/pom.xml

```diff
@@ -1,19 +1,8 @@
-<?xml version="1.0" encoding="UTF-8"?>
 <!--
-  ~ Copyright 2021 The Bank of New York Mellon.
-  ~
-  ~ Licensed under the Apache License, Version 2.0 (the "License");
-  ~ you may not use this file except in compliance with the License.
-  ~ You may obtain a copy of the License at
-  ~
-  ~     http://www.apache.org/licenses/LICENSE-2.0
-  ~
-  ~ Unless required by applicable law or agreed to in writing, software
-  ~ distributed under the License is distributed on an "AS IS" BASIS,
-  ~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-  ~ See the License for the specific language governing permissions and
-  ~ limitations under the License.
-  -->
+ Copyright 2021 BNYMellon. All rights reserved.
+ Use of this source code is governed by a BSD-style
+ license that can be found in the LICENSE file.
```

Comment:
I might have missed but I dont see a LICENSE file added to the PR. I understand that the Apache License file is deleted, but we will need a BSD license file.